### PR TITLE
Revert "Actually add Compute Library tests to the Jenkins File (#8394)"

### PR DESCRIPTION
**Pull request by u99127**
This reverts commit a00d21156ea749d65c9a59b3a79dac501723086b till we
can understand why this is failing in CI but not on developer machines.

@tqchen @leandron @masahi 

Sorry about the breakage but here's a revert if that can be merged asap.

### CODE REVIEW

**Comment by leandron** [comment]
I'm not 100% sure that #8394 needs to be reverted, considering that there is evidence of the tests passing today: https://ci.tlcpack.ai/blue/organizations/jenkins/tvm/detail/PR-8402/3/pipeline.

It might as well be a transient. Needs investigation, but maybe doesn't justify reverting that PR as a whole.

**Comment by u99127** [comment]
I did see another of my PRs fail yesterday with the same sort of failure -.https://ci.tlcpack.ai/blue/organizations/jenkins/tvm/detail/PR-8400/1/pipeline

I've tried respinning that one.

Lets hope it is transient 🤞

**Comment by u99127** [comment]
I think this should be dropped as the CI appears to be holding.

**Comment by u99127** [comment]
Lets not merge this if CI holds - I'll leave it here for a day or so before Closing this out.

**Comment by masahi** [comment]
I've hit the failure today https://ci.tlcpack.ai/blue/organizations/jenkins/tvm/detail/PR-8366/21/pipeline/258

**Comment by electriclilies** [comment]
I also hit the failure today https://ci.tlcpack.ai/blue/organizations/jenkins/tvm/detail/PR-7518/79/pipeline/

**Comment by tqchen** [comment]
we would require the jenkinsfile to be in main in order to take effect. please send a PR to first cleanup the arm compute_lib.sh to be nop, then we can merge and delete the step

**Comment by tqchen** [comment]
tracking issue https://github.com/apache/tvm/issues/8417

**Review by jroesch — APPROVED** [review]

**Pull request merged**
